From 27a7f0020b0965a83559de04673551cf92eb4cbc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Thu, 7 Jan 2016 08:54:35 +0100
Subject: [PATCH] Respect enviroment CFLAGS
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Configure fails if LDFLAGS carries -pie and CFLAGS -fPIC. That's
because confifgure.ac resets CFLAGS.

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 960a618..bb33774 100644
--- a/configure.ac
+++ b/configure.ac
@@ -53,7 +53,7 @@ AS_IF([test "x${prefix}" = "xNONE"], [
 # ================
 # Check for cflags
 # ================
-CFLAGS="-D_GNU_SOURCE -Wall -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64"
+CFLAGS="${CFLAGS} -D_GNU_SOURCE -Wall -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64"
 AC_ARG_ENABLE([werror],
     [AS_HELP_STRING([--enable-werror], [Treat all warnings as errors, useful for development])],
     [enable_werror="$enableval"],
-- 
2.5.0

